src/rt/cover.c

```diff
@@ -162,6 +164,23 @@ bool cover_is_stmt(tree_t t)
    }
 }
 
+bool cover_skip_array_toggle(cover_tagging_t *tagging, int a_size)
+{
+   assert (tagging);
+
+   // Array is equal to or than configured limit
+   if (cover_enabled(tagging, COVER_MASK_TOGGLE_IGNORE_ARRAYS_FROM) &&
```

Comment:
This could be `if (tagging->array_limit != 0 && a_size >= tagging->array_limit)` and then you don't need this enum value.